src/platforms/stm32/src/lib/gpio_driver.c

```diff
@@ -229,6 +229,19 @@ static term create_pair(Context *ctx, term term1, term term2)
     return ret;
 }
 
+static term reason_atom_to_error_tuple_safely(Context *ctx, term err_atom)
```

Comment:
this function name sounds a bit too much. What about `error_tuple_maybe_gc`?
Short and clear about what it does.